microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -20,145 +20,184 @@ ms.date: 05/04/2021
 
-1. Press `Ctrl`+`P` (Windows, Linux) or `Command`+`P` (macOS).  The **Open File** dialog opens.
+1. In a new window or tab, go to a webpage, such as the [Accessibility-testing features demo](https://microsoftedge.github.io/Demos/devtools-a11y-testing/).  (The source files for that webpage are at [MicrosoftEdge / Demos > devtools-a11y-testing](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-a11y-testing).)
 
-   ![The Open File dialog.](../media/resources-command-menu-empty.msft.png)
+1. To open DevTools, right-click the webpage, and then select **Inspect**. Or press `Ctrl`+ `Shift`+ `I` (Windows, Linux) or `Command`+ `Option`+ `I` (macOS).
 
-1. Select the file from the dropdown list, or start typing the filename and press `Enter` once the correct file is highlighted in the autocomplete box.
+1. When DevTools has focus, press `Ctrl`+ `P` (Windows, Linux) or `Command`+`P` (macOS). Or, you can click the (![Customize icon](../media/customize-devtools-icon-light-theme.png)) menu button in DevTools and then select **Open file**.  The Command Menu opens, displaying the **Open File** list:
 
-   ![Type a filename in the Open File dialog.](../media/resources-command-menu-file-search.msft.png)
+   ![The Command Menu, displaying the Open File list](../media/resources-command-menu-empty.msft.png)
+
+1. If there's a greater-than (>) prompt, press `Backspace` to get to the **Open File** prompt.
+
+1. Start typing the filename, and then press `Enter` when the correct file is highlighted in the autocomplete box, or select the file from the dropdown list:
+
+   ![Typing part of a filename in the Open File list of the Command Menu](../media/resources-command-menu-file-search.msft.png)
 
 
 <!-- ====================================================================== -->
-## Open resources in the Network tool
+## Browse resource files in the Network tool
+
+In the **Network** tool, you can inspect the resource files that make up the current webpage, such as `.html`, `.css`, `.js`, and image files.  To learn how to get the details for a specific resource, see [Inspect the details of the resource](../network/index.md#inspect-the-details-of-the-resource) in _Inspect network activity_.
+
+When you go to the **Network** tool and then refresh the page, all the webpage's resource files are listed in the **Network** tool, one row per resource file:
+
+![Webpage resource files listed in the Network Log, one resource file per row](../media/resources-network-resources.msft.png)
 
-See [Inspect the details of a resource](../network/index.md#inspect-the-details-of-the-resource) in _Inspect network activity_.
-<!--why?-->
+When **All** is selected as the resources filter, all the retrieved resource files for the current webpage are listed, in the lower left:
 
-<!-- need lead-in -->
+![Inspect a resource in the Network tool](../media/resources-network-response.msft.png)
```

Comment:
I'd probably enlarge the red box to include the network-tutorial node, since it's what's shown to the right eg index.html listing.
notes:
the list of filenames (including probably "network-tutorial/" node) in lower left ( = .html file; might be gray area, sometimes it's a resource, practically). By default, that top node is selected and the .html file is listed, so seems good to call it a resource that gets filtered.

Reply:
done